pkg/trace/config/config.go

```diff
@@ -252,6 +252,18 @@ type ProfilingProxyConfig struct {
 	AdditionalEndpoints map[string][]string
 }
 
+// EvpIntakeProxy ...
+type EvpIntakeProxy struct {
```

Comment:
```suggestion
type EVPIntakeProxy struct {
```
We generally try to keep acronyms upper case.